app/controllers/spree/admin/base_controller.rb

```diff
@@ -82,6 +82,9 @@ def check_json_authenticity
       private
 
       def html_request?
+        return false if request.format.symbol == :turbo_stream ||
+                        request.format.to_s.include?("turbo")
+
```

Comment:
You'd think checking for `:turbo_stream` would be enough ? but I guess not. So I think we could just simplify it to 
```suggestion
        return false if request.format.to_s.include?("turbo")
```
What do you think ?